### DISCUSSION TITLE
Could someone please walk me through this.

### DISCUSSION THREAD

**Discussion by fOLLY709** [rnelsonee/dc-metro#2] [Q&A]
Every time I attempt, it breaks down. Not sure what is going on. I was able to get LED to work for the demo but once I follow these instructions, it breaks.

**Comment by rnelsonee** [comment]
It's possible I left a step out, but the steps are very similar for the upstream projects (which you can also try to make sure those work, or read those discussions for people with the same issue). Note for mine to work you need to copy those *.bdf files into the `lib` folder.
